### PR TITLE
New version: OpenBLAS_jll v0.3.10+2

### PR DESCRIPTION
**Pull request by jlbuild**
Autogenerated JLL package registration

* Registering JLL package OpenBLAS_jll.jl
* Repository: https://github.com/JuliaBinaryWrappers/OpenBLAS_jll.jl
* Version: v0.3.10+2

### CODE REVIEW

File: O/OpenBLAS_jll/Compat.toml [hunk 1]
```diff
@@ -3,3 +3,4 @@ julia = "1"
 
 ["0.3.10-0"]
 JLLWrappers = "1.1.0-1"
+julia = "1.6.0-1"
```
[the remark this answers is not in the document]
Comment (DilumAluthge) [reply O/OpenBLAS_jll/Compat.toml:6]: What version of Registrator.jl generated this pull request?

[the remark this answers is not in the document]
Comment (staticfloat) [reply O/OpenBLAS_jll/Compat.toml:6]: Supposedly 1.2.2

[the remark this answers is not in the document]
Comment (staticfloat) [reply O/OpenBLAS_jll/Compat.toml:6]: Okay I think I figured it out

[the remark this answers is not in the document]
Comment (DilumAluthge) [reply O/OpenBLAS_jll/Compat.toml:6]: That is concerning.

Do you know the version of RegistryTools.jl?

Actually the full manifest would be ideal.

I'll also loop @fredrikekre in on this.

[the remark this answers is not in the document]
Comment (staticfloat) [reply O/OpenBLAS_jll/Compat.toml:6]: https://github.com/JuliaPackaging/Yggdrasil/blob/master/.ci/Manifest.toml

[the remark this answers is not in the document]
Comment (staticfloat) [reply O/OpenBLAS_jll/Compat.toml:6]: I wonder if it's something special about a `0.X.Y` version.

[the remark this answers is not in the document]
Comment (DilumAluthge) [reply O/OpenBLAS_jll/Compat.toml:6]: We should still figure out why the automated pull request didn't do the right thing.

[the remark this answers is not in the document]
Comment (staticfloat) [reply O/OpenBLAS_jll/Compat.toml:6]: Agreed; but I'm going to march along and keep getting JLLs merged.  For the record, this is the Yggdrasil run that created this PR: https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=7087&view=logs&j=12f17990-53ef-5136-c697-70a8cd1d430e&t=47ad8431-de36-5e15-22e3-706b0aec61ee

[the remark this answers is not in the document]
Comment (DilumAluthge) [reply O/OpenBLAS_jll/Compat.toml:6]: You have the latest version of Registrator.jl and RegistryTools.jl.

[the remark this answers is not in the document]
Comment (staticfloat) [reply O/OpenBLAS_jll/Compat.toml:6]: It does appear to happen a lot with version numbers that start with `0`: https://github.com/JuliaRegistries/General/pull/24077/commits/b7a0a1132a419ded4e4d6b2c203428eb0c847a22

[the remark this answers is not in the document]
Comment (DilumAluthge) [reply O/OpenBLAS_jll/Compat.toml:6]: Does this only happen with JLLs?

@KristofferC or @fredrikekre have you ever seen this happening on non JLL PRs?

[the remark this answers is not in the document]
Comment (DilumAluthge) [reply O/OpenBLAS_jll/Compat.toml:6]: And @staticfloat has it ever happened with a JLL that does not have a 0 major version number?

[the remark this answers is not in the document]
Comment (staticfloat) [reply O/OpenBLAS_jll/Compat.toml:6]: I don't know.

**Pull request merged**